### PR TITLE
Auto save avatars

### PR DESCRIPTION
**Pull request by bago2k4**
Card: https://trello.com/c/xDtmKXCC

From: https://paper.dropbox.com/doc/Misc-UX--ANGz0bS9GxPlXU0OdvuNQNMdAg-zmqEXqPjOZGPJJSxUfPxC

Item:
> Once you've chosen a new user avatar you shouldn't have to hit the Save button; consider it already saved.

This was initially thought for mobile user avatar change only. I extended it to both mobile and desktop and to both user and org avatar since both have the same UI where the save button is down below other fields and the avatars feel disconnected from it.

To test:
- go to org settings
- change the org avatar
- [x] do you see a notification when the upload finishes? Good
- [x] did the avatar changed in the navigation bar too? Good
- [x] can you edit data inside the 3 settings tabs w/o problems? Good
- now stop storage service and try to change the avatar again
- [x] do you get a notification about the error? Good
- [x] do you get the old avatar back in the view? Good
- switch to mobile
- restart storage service
- go to user profile from the menu
- change your avatar
- [x] do you see a notification when the upload finishes? Good
- [x] did your avatar changed in the top right menu too? Good
- [x] can you change your first or last name or other data inside the profile panel w/o problems? Good
- [x] can you change data inside the notifications panel w/o problems? Good
- now stop auth service and try to change the avatar again
- [x] do you get a notification about the error? Good
- [x] do you get your old avatar back in the view? Good

### CODE REVIEW

**Comment by bago2k4** [comment]
[Automatic avatar save](https://trello.com/c/xDtmKXCC/1483-automatic-avatar-save)

File: src/oc/web/components/ui/org_avatar.cljs [hunk 1]
```diff
@@ -18,9 +18,9 @@
     (when show-org-avatar?
       [:img.org-avatar-img
         {:src (:logo-url org-data)
-         :style #js {:height (str (:logo-height org-data) "px")
-                     :marginTop (when (< (:logo-height org-data) default-max-logo-height)
-                                 (str (/ (- default-max-logo-height (:logo-height org-data)) 2) "px"))}
+         ; :style #js {:height (str (:logo-height org-data) "px")
```
Comment (thepug) [inline src/oc/web/components/ui/org_avatar.cljs]: can this code just be removed?

**Comment by thepug** [comment]
Unable to change org data after uploading org icon.

http://recordit.co/3dJymWpxDN

**Comment by bago2k4** [comment]
@thepug fixed, ready for review again

**Pull request merged**